Makefile

```diff
@@ -4,6 +4,7 @@
 
 # Use bash for inline if-statements in arch_patch target
 SHELL:=bash
+ARCH:=$(shell uname -m)
```

Comment:
I don't think this line is needed.
It can also be a source of confusion, because make might be run in the host environment, not the target.

Reply:
Good call, thanks I'll drop it